examples/nlp/text_normalization_as_tagging/utils/eval.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+This script can be used to compare the inference output of Thutmose tagger with multi_reference file
+
+USAGE Example:
+  python eval.py \
+     --inference_file= \
+     --reference_file= \
+     --print_other_errors
+
+The inference file is a tsv file in which the first column contains the predicted sentence text.
+The reference file is a tsv file in which
+    the first column contains the input sentence text,
+    the second column contains the reference sentence text (taken from Google TN dataset)
+    the third column (optional) contains additional acceptable references for semiotic spans in this sentence.
+    E.g.
+        mizoguchi akiko september twenty ten    mizoguchi akiko september 2010     DATE 2 5 | sept 2010 | sep. 2010 ...
```

Comment:
what is `DATE 2 5`?

Reply:
add a reference to a script to generate additional references?

Reply:
done